resources/benchmark-runner.mjs

```diff
@@ -323,6 +333,9 @@ export class BenchmarkRunner {
         this._page = null;
         this._metrics = null;
         this._iterationCount = params.iterationCount;
+        if (params.shuffleSeed != "off") {
+            this._rng = seededHashRng(params.shuffleSeed);
```

Comment:
Since rng is an abbreviation, we should probably use all caps as in this._RNG and SeededHashRNG.
Better yet, just spell out RandomNumberGenerator so:
```
this._suiteOrderRandomNumberGenerator = seededHashRandomNumberGenerator(a);
```